src/scenarios/shared/mauisharedpython.py

```diff
@@ -93,4 +94,80 @@ def install_versioned_maui(precommands: PreCommands):
         dump_dict_to_json_file(rollback_dict, f"rollback_{target_framework_wo_platform}.json")
         workload_install_args += ['--from-rollback-file', f'rollback_{target_framework_wo_platform}.json']
 
-    precommands.install_workload('maui', workload_install_args) 
+    precommands.install_workload('maui', workload_install_args)
+
+# TODO: the feed should be updated to the latest
```

Comment:
Once we have darc setup to update NuGet feeds, this won't be needed anymore, correct?

Reply:
I wrote a new logic which extracts the latest feed from the repo's Nuget.config so we don't need this anymore.